### ISSUE TITLE
Support for Shelly Plus Plug S and more ?

### ISSUE THREAD

**Issue by marcel255** [alexryd/homebridge-shelly-ng#66]
Hello, can you please add these Shelly´s for support, most importantly Shelly Plus Plug S  :)

Shelly Plus Plug S
```{"name":null,"id":"shellyplusplugs-3ce90e303dc0","mac":"3CE90E303DC0","model":"SNPL-00112EU","gen":2,"fw_id":"20230209-131834/0.13.0-g68ba560","ver":"0.13.0","app":"PlusPlugS","auth_en":false,"auth_domain":null}```

Shelly Pro 3EM
```{"name":null,"id":"shellypro3em-ec62608912d0","mac":"EC62608912D0","model":"SPEM-003CEBEU","gen":2,"fw_id":"20230209-154545/0.13.1-g27ee371","ver":"0.13.1","app":"Pro3EM","auth_en":false,"auth_domain":null}```

Shelly Plus Smoke
```{"name":null,"id":"shellyplussmoke-80646fd06084","mac":"80646FD06084","model":"SNSN-0031Z","gen":2,"fw_id":"20230209-131913/0.13.0-g68ba560","ver":"0.13.0","app":"PlusSmoke","auth_en":false,"auth_domain":null}```

Thanks !

**Comment by vschroederdo** [comment]
Hello, please add thes Shelly Plus Plug S.

**Comment by TP369** [comment]
Yes, please! I'd love to use my new Shelly Plus Plug S (EU) with Homekit.

**Comment by matko1974** [comment]
Same here. Missing the Plus Plug S!

**Comment by Heinz7000** [comment]
Same ehre. Missing the Plus Plug S and the Pro 3EM.

Heinz

**Comment by platini76** [comment]
same here!!!

**Comment by Kasper05** [comment]
is there any progress on this.

**Comment by Stromberg73** [comment]
Same here:
Shelly Plus Plug S (EU)
Shelly Pro 3EM
Shelly Smoke

And please don´t forget the Shelly Plus Addon (i.e: Shelly plus 1 with addon plus in a garage door with magnetic sensor, in order to know if door is opened or closed) so we can see the door status in Homekit

Thank you in advance

**Comment by fabiovit** [comment]
Also Shelly Plus Plug IT.

**Comment by hatty24** [comment]
Hello,

is anyone else working on this project?

If so, it would be great if the new shellys, mainly the Plus Plug S would be implemented.

Thanks a lot

**Comment by geins** [comment]
There is a pull request in node module shellies-ng, which this package is using, but not committed since 4m: https://github.com/alexryd/node-shellies-ng/pull/2
paddek:support-more-devices

**Comment by Kasper05** [comment]
????????

**Comment by oreus2** [comment]
Hello, please add the Shelly Plus Plug S (EU).

**Comment by hatty24** [comment]
I think that since Shelly has announced a Matter update for the Plus and Pro series, with which you can integrate the Shellys directly into Homekit, nothing more will happen here.

**Comment by Hr-Frodo** [comment]
is it possilbe to add this device?

i do not have a Homepod or something else and like to use the devices also.

**Comment by keptenkrems** [comment]
Hi, can you please add the Plug, as in this post written, the next Version (gen3) will be have matter Support. The Version Shelly Plus Plug S does Not get it, isn‘t? Thanks in Advance.

https://www.smarthomeassistent.de/shelly-plug-s-mtr-gen3-neue-smarte-steckdose-mit-matter/
